examples/contracts/src/lib.rs

```diff
@@ -273,47 +273,46 @@ mod tests {
             "../../packages/fuels/tests/contracts/contract_test/out/debug/contract_test.bin",
             LoadConfiguration::default(),
         )?
-        .deploy(&wallet, TxParameters::default())
+        .deploy(&wallet, TxPolicies::default())
         .await?;
 
         println!("Contract deployed @ {contract_id}");
-        // ANCHOR: tx_parameters
+        // ANCHOR: tx_policies
         let contract_methods = MyContract::new(contract_id.clone(), wallet.clone()).methods();
 
-        let my_tx_parameters = TxParameters::default()
+        let tx_policies = TxPolicies::default()
             .with_gas_price(1)
-            .with_gas_limit(1_000_000)
```

Comment:
Well `gas_limit` is not the limit for the whole tx anymore. IHMO it makes sense to rename it so that the user immediately knows that this is something new.

Reply:
I get it. However:

> Well gas_limit is not the limit for the whole tx anymore

Coming from other blockchain ecosystems/platforms, this sentence alone is confusing. I'm trying to put myself in the user's shoes here. When I set the gas limit for a transaction, that's it. Having to think about multiple gas limits is confusing. I'm still going through the changes so I don't know this: will the user ever be required to set _both_ gas limits in a single transaction?

Reply:
The sdk is just reflecting the changes from `fuel-core` and `fuel-vm`. `script_gas_limit` now limits only script gas. and `max_fee` limits the whole `tx`. The user might set both limits in one `tx`. cc @xgreenx